viz-lib/src/visualizations/details/DetailsRenderer.jsx

```diff
@@ -37,16 +38,13 @@ export default function DetailsRenderer({ data }) {
 
   return (
     <div>
-      <table className="table table-bordered details-viz">
-        <tbody>
-          {map(columns, key => (
-            <tr key={key}>
-              <th>{key}</th>
-              <td>{renderValue(row[key], types[key])}</td>
-            </tr>
-          ))}
-        </tbody>
-      </table>
+      <Descriptions className="details-viz" size="small" column={1} bordered>
+        {map(columns, key => (
+          <Descriptions.Item key={key} label={key}>
+            {renderValue(row[key], types[key])}
+          </Descriptions.Item>
+        ))}
+      </Descriptions>
```

Comment:
Only thing I noticed is that we need some spacing between the table & the paginator.